Java/parkservices/src/test/java/com/amazon/randomcutforest/parkservices/AnomalyDescriptorTest.java

```diff
@@ -84,9 +84,15 @@ public void PastValuesTest(ScoringStrategy strategy) {
                         if (firstResult.getRelativeIndex() == 0) {
                             assertArrayEquals(firstResult.getPastValues(), firstResult.getCurrentInput(), 1e-10);
                         }
-                        if (firstResult.isReasonableForecast()) {
-                            assertNotNull(firstResult.getExpectedValuesList());
-                            assertNotNull(firstResult.getExpectedValuesList()[0]); // the expected value
+                        if (firstResult.getExpectedValuesList() != null
+                                && firstResult.getExpectedValuesList()[0] != null) {
+                            assert (firstResult.isReasonableForecast());
```

Comment:
Should you use assertTrue? Assertions are not enabled by default in Java 18, or in any version of Java. The JVM disables assertion validation by default for backward compatibility. If you want to use assertions, they must be explicitly enabled using either the -enableassertions command line argument or its shorthand -ea​1​.

Reply:
Thanks. Changing.